### ISSUE TITLE
other example applications

### ISSUE THREAD

**Issue by wenjing** [twister2016/twister#51]
please follow the spirit similar to #48. this refers to examples " gateway and multiport.
by the way, multiport should be a general feature, rather than a separate application. right?

**Comment by chaudhryusama** [comment]
I agree with the statement that it should be a general feature ( and it is  a general feature). The only inspiration of multiport was to test the Twister to run on more than one port.
